samples/suit/smp_transfer/README.rst

```diff
@@ -0,0 +1,382 @@
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -DFILE_SUFFIX=bt -SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
+
+         .. note::
+
+            |application_sample_long_path_windows|
+
+            In this case, you may need to run the following instead:
+
+            .. code-block:: console
+
+               west build -p -b --sysbuild nrf54h20dk/nrf54h20/cpuapp -d C:/ncs-lcs/work-dir -- -DFILE_SUFFIX=bt -SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
+
+         Another :file:`root.suit` file is created after running this command, that contains the updated firmware.
+         You must manually transfer this file onto the same mobile device you will use with the nRF Device Manager app.
+
+   .. group-tab:: Over UART
+
+      1. Open a terminal window in |sample path|.
+      #. Build the sample:
+
+         .. code-block:: console
+
+             west build -p -b --sysbuild nrf54h20dk/nrf54h20/cpuapp
+
+         .. note::
+
+            |application_sample_long_path_windows|
+
+            In this case, you may need to run the following instead:
+
+            .. code-block:: console
+
+               west build -p -b --sysbuild nrf54h20dk/nrf54h20/cpuapp -d C:\ncs-lcs\west_working_dir\build\
+
+         If you want to further configure your sample, see :ref:`configure_application` for additional information.
+
+         After running the ``west build`` command, the output build files can be found in the :file:`build/dfu` directory.
+         The output build files can be found in the :file:`build/DFU` directory, including the :ref:`app_build_output_files_suit_dfu`.
+         For more information on the contents of the build directory, see :ref:`zephyr:build-directory-contents` in the Zephyr documentation.
+         For more information on the directory contents and structure provided by Sysbuild, see :ref:`zephyr:sysbuild` in the Zephyr documentation..
+
+      #. Connect the DK to your computer using a USB cable.
+      #. Power on the DK.
+      #. Program the sample to the kit (see :ref:`programming_cmd` for instructions).
+
+         .. note::
+
+            |application_sample_long_path_windows|
+
+            In this case, you may need to run the following instead:
+
+            .. code-block:: console
+
+               west flash --erase -d C:/ncs-lcs/work-dir
+
+      #. Update the SUIT envelope sequence number, by rebuilding the sample with an updated sequence number:
+
+         .. code-block:: console
+
+            west build -p -b nrf54h20dk/nrf54h20/cpuapp --sysbuild -- -SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
+
+         .. note::
+
+            |application_sample_long_path_windows|
+
+            In this case, you may need to run the following instead:
+
+            .. code-block:: console
+
+               west build -p -b nrf54h20dk/nrf54h20/cpuapp -d C:/ncs-lcs/work-dir --sysbuild -- -SB_CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
```

Comment:
Hm, would it be possible to post this note at the beginning of the procedure once, with some generic information, instead of repeating it 5 times within the same procedure? I know the command is different each time, but also what is critical for this information -- `-d <path>` -- stays the same.